src/main/java/org/thoughtslive/jenkins/plugins/jira/api/Fields.java

```diff
@@ -27,7 +27,10 @@
 @Builder
 public class Fields implements Serializable {
 
-  private static final long serialVersionUID = -6758862154743675930L;
+  private static final long serialVersionUID = 9022728696230161151L;
+
+  @JsonProperty("parent")
+  private Parent parent;
```

Comment:
This can be type of existing class `BasicIssue`[1], then we don't need to add the following new class, `Parent`.

Otherwise this change looks good, Thanks for your contribution @abeeskau

[1] https://github.com/jenkinsci/jira-steps-plugin/blob/master/src/main/java/org/thoughtslive/jenkins/plugins/jira/api/input/BasicIssue.java